src/subscription.ts

```diff
@@ -503,20 +548,33 @@ export class Subscription extends EventEmitter {
    *   const apiResponse = data[1];
    * });
    */
-  get(gaxOpts, callback) {
-    if (is.fn(gaxOpts)) {
-      callback = gaxOpts;
-      gaxOpts = {};
-    }
+  get(callback: CreateSubscriptionCallback): void;
+  get(gaxOpts?: CallOptions&
+      GetCallOptions): Promise<CreateSubscriptionResponse>;
```

Comment:
Can we just have `GetCallOptions` extend `CallOptions` so you don't need to do the union?

Reply:
done